### DISCUSSION TITLE
关于已上架商店的插件修改问题

### DISCUSSION THREAD

**Discussion by jks15satoshi** [nonebot/discussions#89] [问答] [answered]
1. 如需对目前已上架至商店的插件信息进行修改（例如仓库重定向、下架商店等），是否需要自行修改 [plugins.json](https://github.com/nonebot/nonebot2/blob/master/website/static/plugins.json) 文件？
2. 如 1 满足，修改变更应当提交至哪个分支？

**Comment by mnixry** [comment, marked as answer]
> 1. 如需对目前已上架至商店的插件信息进行修改（例如仓库重定向、下架商店等），是否需要自行修改 [plugins.json](https://github.com/nonebot/nonebot2/blob/master/website/static/plugins.json) 文件？

建议自行PR修改
我们可能会在最近工作告一段落后对插件进行批量检查，筛出一些版本不适配和不再更新的插件进行Tag标注或者移除

> 2. 如 1 满足，修改变更应当提交至哪个分支？

建议提交到master

**Reply by jks15satoshi** [reply]
Got it.
